@types/interfaces.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface TimePickerTime {
+    hours: number;
+    minutes: number;
+    seconds: number;
+    milliseconds: number;
+}
```

Comment:
I would suggest the following.
```suggestion
export interface TimePickerTime {
  hours: string | number;
  minutes: string | number;
  seconds?: string | number;
  milliseconds?: string | number;
}
```

`seconds` and `milliseconds` are optional (for `formatTime`) and it works both with numbers and strings. However `parseTime` returns the numbers as strings.

Reply:
For example:
```js
timePicker.i18n.parseTime('03:00');
// returns:
{hours: "03", minutes: "00", seconds: undefined, milliseconds: undefined}
```

Reply:
Actually since `seconds` and `milliseconds` may be explicitly set to `undefined` and not just left out of props, maybe it's more correct to use?:
```ts
export interface TimePickerTime {
  hours: string | number;
  minutes: string | number;
  seconds?: string | number | undefined;
  milliseconds?: string | number | undefined;
}
```

Reply:
I just tested a bit in TS playground and it seems to be valid to pass `undefined` to an optional property even if `undefined` is not explicitly included. So no need to add `| undefined` here.

Reply:
Looks like you're right... This just makes the DX for writing `formatTime` really bad, because you can't know if the parameter `time.hours` etc. is a `string` or a `number`.

Reply:
Suggestion committed.